apps/menu-bar/modules/web-authentication-session/ios/WebAuthContextProvider.swift

```diff
@@ -0,0 +1,13 @@
+import AuthenticationServices
+
+public class WebAuthContextProvider: NSObject, ObservableObject, ASWebAuthenticationPresentationContextProviding {
+  public func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
+    var anchor: ASPresentationAnchor?
+
+    DispatchQueue.main.sync {
+      anchor = NSApp.mainWindow!
+    }
+
+    return anchor!
```

Comment:
```suggestion
    var anchor: ASPresentationAnchor?

    DispatchQueue.main.sync {
      anchor = NSApp.mainWindow
    }

    return anchor ?? ASPresentationAnchor()
```
We can create our own instance and avoid force unwrapping. It should never happen that the mainWindow is nil but I think this is safer.